note.txt

```diff
@@ -0,0 +1,64 @@
+switch branch: git checkout fix/kelvin-setup
```

Comment:
Actually, you should checkout the `main` branch, and create your own feature branch.